pom.xml

```diff
@@ -244,7 +244,7 @@
         <dependency>
         	<groupId>org.mitre.hapifhir</groupId>
         	<artifactId>smart-backend-auth</artifactId>
-        	<version>0.0.3</version>
+        	<version>0.0.2</version>
```

Comment:
Why is the version number decreasing? If we have 0.0.3 and we're adding new things, it should become 0.0.4. The place to update the version number in the library is here: https://github.com/mcode/smart-backend-auth/blob/master/build.gradle#L57 (though I'd also suggest changing the examples readme to include the most recent version number as well)

Reply:
I think I accidentally branched from another branch which still has 0.0.1. That might explain why I was having issues getting publish to maven local to work. I will bump this to 0.0.4 here and in the medmorph-fhir-server pom.xml

Reply:
Just realized this it the fhir-server PR.... so I will update here and in the library